src/main/java/org/prebid/server/spring/config/ServiceConfiguration.java

```diff
@@ -234,10 +234,17 @@ UidsCookieService uidsCookieService(
             @Value("${host-cookie.family:#{null}}") String hostCookieFamily,
             @Value("${host-cookie.cookie-name:#{null}}") String hostCookieName,
             @Value("${host-cookie.domain:#{null}}") String hostCookieDomain,
-            @Value("${host-cookie.ttl-days}") Integer ttlDays) {
+            @Value("${host-cookie.ttl-days}") Integer ttlDays,
+            @Value("${host-cookie.max-cookie-size-bytes}") Integer maxCookieSizeBytes) {
+
+        final int minCookieSizeBytes = 500;
```

Comment:
Would like to suggest to move this validation to `UidsCookieService`'s constructor.